chapter-07/wip/Engine/Sound/SoundManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using chapter_07.Engine.States;
+
+using Microsoft.Xna.Framework.Audio;
+using System;
+using System.Collections.Generic;
+
+namespace chapter_07.Engine.Sound
+{
+    public class SoundManager
+    {
+        private int _soundtrackIndex = -1;
+        private List<SoundEffectInstance> _soundtracks = new List<SoundEffectInstance>();
+        private Dictionary<Type, SoundEffect> _soundBank = new Dictionary<Type, SoundEffect>();
+
+        public void AddSoundtrack(List<SoundEffectInstance> tracks)
```

Comment:
Since this is really placing the soundtrack, maybe change the name to **SetSoundtrack** instead?

Reply:
good call :)